railties/test/generators/app_generator_test.rb

```diff
@@ -1009,7 +1009,7 @@ def test_skip_webpack_install
     assert_no_file "config/webpacker.yml"
 
     output = Dir.chdir(destination_root) do
-      `bin/rails help`
+      `bin/rails -h`
```

Comment:
`bin/rails help` no longer outputs the extended commands. So this had to be changed to `bin/rails -h` which does output the extended commands.